tensorflow_hub/tools/make_image_classifier/make_image_classifier.py

```diff
@@ -185,7 +188,7 @@ def main(args):
     _set_gpu_memory_growth()
 
   model, labels, train_result = lib.make_image_classifier(
-      FLAGS.tfhub_module, image_dir, hparams, FLAGS.image_size)
+      FLAGS.tfhub_module, image_dir, hparams, FLAGS.summaries_dir, FLAGS.image_size)
```

Comment:
After the change in lib.py, the argument order here looks wrong now. Please fix this and run to test it.

Reply:
Guess I should look for a decent Python IDE which supports refactoring :-)